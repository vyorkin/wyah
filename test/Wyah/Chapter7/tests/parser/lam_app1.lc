let f = (\x -> x - 1) 2;
