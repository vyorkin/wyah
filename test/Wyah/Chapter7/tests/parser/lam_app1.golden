Program [Decl "f" (EApp (ELam (Var "x") (EOp Sub (EVar (Var "x")) (ELit (LInt 1)))) (ELit (LInt 2)))]